src/pipit/settings/prod.py

```diff
@@ -26,6 +25,12 @@
             "MAX_ENTRIES": 1000,
         },
     },
+    "holon_cache": {
+        "BACKEND": "holon.cache.holon_database_cache.HolonDatabaseCache",
+        "LOCATION": "holon_cache",
+        "TIMEOUT": None,
+        "OPTIONS": {"MAX_ENTRIES": 100000},
+    },
```

Comment:
Is er een bewuste keuze gemaakt voor de database cache vs alternatieven? Misschien goed om dit nog wel een keer te doen zodat we niet op lange termijn tegen limieten aan gaan lopen

Reply:
Het idee is dat het persistent is, dus dat er bij een uitval/reset niet een nacht aan rekenwerk verloren gaat. Ik weet verder niet of er andere persistent alternatieven zijn. Denk je dat we op de korte termijn tegen limieten aan zouden kunnen lopen?